UndertaleModLib/UndertaleData.cs

```diff
@@ -384,6 +384,22 @@ public class UndertaleData : IDisposable
         /// </summary>
         public bool GMLCacheIsReady { get; set; } = true;
 
+        /// <summary>
+        /// An array of a <see cref="UndertaleData"/> properties with <see cref="IList{T}"/> type.
+        /// </summary>
+        public PropertyInfo[] ListProperties { get; private set; }
```

Comment:
Name is IMO a bit confusing. Based solely on the name, this sounds like this is a *list* of *all* Properties in UnderaleData.
This is neither a list tho, nor does it contain all properties.
Maybe name it something along the lines of `AllListProperties` ?

Reply:
If this was a list of all properties, then it would be named `Properties` or `PropertiesList`.
Also, there's a description for it.

So, I'm not sure if it should be 'AllListProperties` or not.

Reply:
> If this was a list of all properties, then it would be named `Properties` or `PropertiesList`

This is heavily context dependant.

> Also, there's a description for it.

You don't always see the description. The name should be understandable enough on its own to a certain extent.